src/DIRAC/DataManagementSystem/Client/DataManager.py

```diff
@@ -1444,8 +1451,9 @@ def __removeCatalogReplica(self, replicaTuples):
             for lfn in success:
                 log.debug("Successfully removed replica.", lfn)
 
-        oDataOperation.setValueByKey("RegistrationOK", len(success))
-        gDataStoreClient.addRegister(oDataOperation)
```

Comment:
`gDataStoreClient.addRegister()` and `gDataStoreClient.commit()` are not the same thing, you can't "translate" them both with `DataOperationSender.sendData()`